src/Sylius/Component/Shipping/Model/ShippingMethod.php

```diff
@@ -78,6 +78,9 @@ public function __construct()
      */
     public function __toString()
     {
+        if(null === $this->translate()->__toString()) {
```

Comment:
If we got `ShippingMethodTranslation::__toString()` fixed we shouldn't care about it here.